src/main/java/edu/harvard/iq/dataverse/dataaccess/FileAccessIO.java

```diff
@@ -656,4 +659,28 @@ private String stripDriverId(String storageIdentifier) {
         }
         return storageIdentifier;
     }
+    
+    //Confirm inputs are of the form of a relative file path that doesn't contain . or ..
+    protected static boolean isValidIdentifier(String driverId, String storageId) {
+        String pathString = storageId.substring(storageId.lastIndexOf("//") + 2);
+        String basePath = "/tmp/";
+        try {
+            String rawPathString = basePath + pathString;
+            Path normalized = Paths.get(rawPathString).normalize();
+            if(!rawPathString.equals(normalized.toString())) {
+                logger.warning("Non-normalized path in submitted identifier " + storageId);
+                return false;
+            }
+            System.out.println(normalized.getFileName().toString());
```

Comment:
Is this println line here on purpose, or were you planning to remove it before this is merged? (just checking; there's a couple more in the PR).

Reply:
Good catch - changed the 2 in the main code to logger.fine (left the one in the test).